jquery.signaturepad.js

```diff
@@ -528,14 +528,14 @@ function SignaturePad (selector, options) {
     }
 
     if (typeItNumChars > oldLength && typed.outerWidth() > element.width) {
-      while (typed.outerWidth() > element.width) {
+      while (typeItCurrentFontSize > 4 && typed.outerWidth() > element.width) {
         typeItCurrentFontSize--
         typed.css('font-size', typeItCurrentFontSize + 'px')
       }
     }
 
     if (typeItNumChars < oldLength && typed.outerWidth() + edgeOffset < element.width && typeItCurrentFontSize < typeItDefaultFontSize) {
-      while (typed.outerWidth() + edgeOffset < element.width && typeItCurrentFontSize < typeItDefaultFontSize) {
+      while (typeItCurrentFontSize < 512 && typed.outerWidth() + edgeOffset < element.width && typeItCurrentFontSize < typeItDefaultFontSize) {
```

Comment:
I only vaguely understand the scenario this PR is trying to fix, but the impression I got was that it was a problem for shrinking the font size. Is it also an issue for large font sizes as well?

Reply:
The original problem is that changing the font size has no impact at all on the typed.outerWidth() causing the long looping. I thought that there should be some kind of guard here too. Maybe 512 is not a good number here or perhaps there is a better solution?

Reply:
Okay, sounds good. I think this is probably fine for now.